addon/lib/templates.js

```diff
@@ -71,3 +71,24 @@ module.exports.feedback = `
   </div>
   <div class="clear"></div>
 </div>`;
+
+module.exports.installed = `
+<div id="browser-doorhanger">
+  <div class="copter-wrapper">
+    <div class="copter fly-down"></div>
+  </div>
+  <p>Test Pilot lets you test out experimental features in Firefox. Ready for takeoff?</p>
```

Comment:
These strings specifically are a part of the addon, which we are not using l20n with yet.